src/app/svg_config.py

```
@@ -12,6 +12,14 @@
 # ---
 load_dotenv(_env_file_path)
 # ---
+CopySvgTranslate_PATH = os.getenv("CopySvgTranslate_PATH", "")
+# ---
+try:
+    import CopySvgTranslate
```

Flake8 (7.3.0)

[error] 'CopySvgTranslate' imported but unused

(F401)